docs/src/content/docs/book/message-mode.mdx

```diff
@@ -11,7 +11,7 @@ It's possible to use raw [`Int{:tact}`][int] values and manually provide them fo
 
 Mode value | Constant name                 | Description
 ---------: | :---------------------------- | -----------
-$0$        | -                             | Ordinary message (default).
+$0$        | `SendDefaultMode{:tact}`      | Ordinary message (default). <Badge text="Available since Tact 1.6" variant="tip" size="medium"/><p/>
```

Comment:
```suggestion
$0$        | <Badge text="Available since Tact 1.6" variant="tip"/> 
 `SendDefaultMode{:tact}`      | Ordinary message (default).
```

Cool, but I'd do this, since the mode was there all this time, but the constant has just been introduced

Reply:
And the `import { Badge } from '@astrojs/starlight/components';` has to be added after the frontmatter (`---` stuff)